### ISSUE TITLE
"Invoice is not a proper lightning payment request" - coinos.io invoices do not work with Bitfinex withdrawals

### ISSUE THREAD

**Issue by jerzybrzoska** [asoltys/coinos#16]
When I generate a lightning invoice at [coinos.io](https://coinos.io
) and paste it into the invoice field at Bitfinex's withdrawal page I get the following:

![bitfinexWitdraw_2020-11-27_16-30-32](https://user-images.githubusercontent.com/51721153/100467860-886cda80-30d3-11eb-9daa-89d59f467925.png)

**Comment by asoltys** [comment]
Looks like you have to specify an amount in the invoice

**Comment by jerzybrzoska** [comment]
Finally it dawned on me. I was specifying the amount earlier, like so 0.0014. The wallet does not seem to understant floats, it only accepts integers.  I was trying to specify the amount in BTC and not in Satoshis, like so: 
![amountCoinos_2020-11-27_21-22-32](https://user-images.githubusercontent.com/51721153/100482287-32aa2980-30f7-11eb-96a5-aef9a05fa228.png)

but to no avail. Likewise when I introduced 2.0014 it rounded it to 2 BTC. I was introducing the amount with my keybooard and not with a mouse. I noticed that your GUI lacks comma. Is it so for the very same reason?
